editor.planx.uk/src/@planx/components/List/schemas/Adverts.ts

```diff
@@ -0,0 +1,40 @@
+import { Schema } from "@planx/components/List/model";
+
+export const ProposedAdvertisements: Schema = {
+  type: "Proposed advertisements",
```

Comment:
nit: this piece of text will be read into the title of each item & the "+ Add another ..." button - so it's actually best if it's singular rather than plural !